app/src/main/java/com/example/cmput301project/User.java

```diff
@@ -1,6 +1,6 @@
 package com.example.cmput301project;
 
-import com.example.cmput301project.ItemClasses.ItemList;
+import com.example.cmput301project.itemClasses.ItemList;

```

Comment:
Java naming condition dictates that all directory names should be in lower case. I left one more of these comments but there are several other places where the package name should be fixed.